### PR TITLE
Requiring a valid user

### PR DESCRIPTION
**Pull request by SCdF**
We now require that you are a user with a specific role before we let you write a build.

I also took the opportunity to get rid of Grunt and replace some vague jslinting with eslint.

### CODE REVIEW

File: ddocs/builds/validate_doc_update.js [hunk 1]
```diff
@@ -1,6 +1,10 @@
-function(newDoc, oldDoc, userCtx, secObj) {
+function(newDoc, oldDoc, userCtx) {
   var version = require('views/lib/version').fn(newDoc._id);
 
+  if (!userCtx || userCtx.roles.indexOf('builds-admin') === -1) {
```
Comment (SCdF) [inline ddocs/builds/validate_doc_update.js:4]: This is the main bit really..

File: test/int/ddoc.js [hunk 2]
```diff
@@ -45,135 +78,167 @@ const withBuildInfo = (doc, specificTime) => {
...
+      invalidBuildsUrl = invalidBuildsUrl.toString();
+      before(() => UsersDb.put(invalidBuildsUser)
+          .then(result => {
+            invalidBuildsUser._rev = result.rev;
+          }));
+      after(() => UsersDb.remove(invalidBuildsUser));
+
+      let InvalidBuildsDb = new PouchDB(invalidBuildsUrl);
+
+      it('Users without the builds-admin role cannot write builds', () =>
```
Comment (SCdF) [inline test/int/ddoc.js:204]: …along with this test

File: ddocs/builds/validate_doc_update.js (hunk 1 above)
Comment (dianabarsan) [reply ddocs/builds/validate_doc_update.js:4]: Hardcoding the role is fine by me. 
We could also allow all roles assigned as db admins + the `_admin` role (like in https://github.com/medic/medic/blob/master/ddocs/medic/validate_doc_update.js#L30)

**Review by dianabarsan — APPROVED** [review]
Looks good!

**Pull request merged**